### PR TITLE
chore(windows/resources): Cherry-pick 14.0 help from #4109

### PR DESCRIPTION
**Pull request by darcywong00**
Addresses some of the TODO's in the 3rd point of keymanapp/help.keyman.com#285

This cherry-picks the 14.0 help updates in #4109 from `master` to `beta`.

Contrasting with http://help.keyman-staging.com/products/windows/14.0/
the following pages don't exist on the beta branch

http://help.keyman-staging.com/products/windows/14.0/common/editions

http://help.keyman-staging.com/products/windows/14.0/context/manualactivate
http://help.keyman-staging.com/products/windows/14.0/context/tsactivate

http://help.keyman-staging.com/products/windows/14.0/troubleshooting/hidden

### CODE REVIEW

File: windows/src/desktop/help/basic/config/options.md [hunk 1]
```diff
@@ -34,20 +34,35 @@ To open the Options tab of Keyman Configuration:
...
 -   Simulate AltGr with Ctrl+Alt
 
-    Some hardware keyboards do not have a designated AltGr key (Right
-    Alt key). Some Keyman keyboards require AltGr. Tick this option to
-    allow Ctrl+Alt to be used for AltGr. Untick this option and Ctrl+Alt
-    will not be used for AltGr.
+    Some hardware keyboards do not have a designated <kbd>AltGr</kbd> key
+    (<kbd>Right Alt</kbd> key). Some Keyman keyboards require
+    <kbd>AltGr</kbd>. Tick this option to allow <kbd>Ctrl</kbd> +
+    <kbd>Alt</kbd> to be used for <kbd>AltGr</kbd<>>. Untick this option
```
Comment (mcdurdin) [inline windows/src/desktop/help/basic/config/options.md:44]:
```suggestion
    <kbd>Alt</kbd> to be used for <kbd>AltGr</kbd>. Untick this option
```

File: windows/src/desktop/help/basic/config/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+![](../../desktop_images/tab-support.png)
+
+## Opening Keyman Configuration
+
+To open Keyman Configuration:
+
+1.  Click on the Keyman icon ![](../../desktop_images/icon-keyman.png), on the
+    Windows Taskbar near the clock.
+
+2.  From the Keyman menu, select Configuration....
```
Comment (mcdurdin) [inline windows/src/desktop/help/basic/config/index.md:19]:
```suggestion
2.  From the Keyman menu, select **Configuration...**.
```

File: windows/src/desktop/help/basic/config/options.md [hunk 3]
```diff
@@ -139,16 +159,11 @@ To open the Options tab of Keyman Configuration:
 -   Switch to appropriate On Screen Keyboard/Help automatically when a
     keyboard is selected
 
-    Each time you turn on a Keyman keyboard, Keyman can automatically
-    detect and switch to the Keyman Toolbox tool which is best for that
-    Keyman keyboard: either the On Screen Keyboard or Keyboard Usage.
-    Tick this option to have Keyman automatically detect and display the
-    best view for your Keyman keyboard. Untick this option to prevent
-    Keyman from switching its display automatically.
+    This option is deprecated in Keyman 14.0 and has no function.
```
Comment (mcdurdin) [inline windows/src/desktop/help/basic/config/options.md:162]: We should remove this entirely -- the option has already been removed.

**Review by mcdurdin — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.229-beta](https://keyman.com/downloads/releases/beta/14.0.229)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.230-beta](https://keyman.com/downloads/releases/beta/14.0.230)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.15-alpha](https://keyman.com/downloads/releases/alpha/15.0.15)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.16-alpha](https://keyman.com/downloads/releases/alpha/15.0.16)

**Pull request merged**